Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -147,7 +150,6 @@ public void logError(final String message) {
           return;
         }
 
-        getSession().getConsoleView().print("Error: " + message + "\n", ConsoleViewContentType.ERROR_OUTPUT);
```

Comment:
I can remove it below as well - this was displaying some API usage error information in the user's console. I thought it would be better to only log it is `LOG.warn()` / `LOG.error()`. Let me know which you'd prefer -